scripts/image

```diff
@@ -399,6 +399,8 @@ if [ "$1" = "release" -o "$1" = "mkimage" -o "$1" = "amlpkg" -o "$1" = "noobs" ]
     echo "Creating Amlogic ZIP auto-install package"
     pushd sign > /dev/null
     echo --update_package=/sdcard/$IMAGE_NAME-update.zip > factory_update_param.aml
+    echo --wipe_data >> factory_update_param.aml
+    echo --wipe_cache >> factory_update_param.aml
```

Comment:
I vote yes for the changes you made and I suggest cleaning up Core's updater-script as well. :)